types/priv_validator.go

```diff
@@ -239,7 +258,7 @@ func (privVal *PrivValidator) signBytesHRS(height, round int, step int8, signByt
 	}
 
 	// Sign
-	sig, err := privVal.Sign(signBytes)
+	sig, err := privVal.Signer.Sign(signBytes)
```

Comment:
Signer is an anonymous embedded interface, so the following should work:
`privVal.Sign()`.  Why not use that?

Reply:
yep, mistake. thanks